src/components/researchPortal/ResearchExpressionPlot.vue

```diff
@@ -268,6 +281,9 @@ export default Vue.component("ResearchExpressionPlot", {
 
 			for (let item of processedData) {
 				for (let tpmVal of item.tpmForAllSamples) {
+					if (!!isNaN(tpmVal)){
+						console.log("we missed one");
+					}
 					tpmVal = !!isNaN(tpmVal) ? 0 : tpmVal;
```

Comment:
Number.isNaN() is used above but these are still isNaN(). Also, double !! in this context is also redundant. It doesn't hurt anything, but be aware that it's not necessary for **if** statements

Reply:
Oh, I think I just forgot to remove this while scrubbing older code. AFAIK this is now completely unncessary.